CONTRIBUTING.rst

```diff
@@ -1,107 +1,108 @@
-Contributing to Crate
-=====================
+============
+Contributing
+============
 
-Thanks for considering contributing to crate.
+Thank you for your interest in contributing.
 
-The steps outlined in this document should be seen as guidelines. These are NOT
-strict rules and we won't blame anyone who tries to contribute.  By following
-these things you'll just make our lives a tad bit easier, but if you don't
-follow them we'll still be happy about your contribution.
+This document is a guideline. Don't worry too much about getting everything
+perfect. We are more than happy to work with you on your contribution.
 
-The easiest way to contribute is either by filling issues on `Github
-<https://github.com/crate/crate/issues>`_ or by contributing to our
-documentation.
+Reporting issues it the easiest way to contribute. But we also accept pull
+requests for the code and for the documentation. For more information on how to
+work on the code or documentation, see our `developer guide`_.
 
-For more information on how to do that please refer to the documentation
-section in the ``DEVELOP.rst``.
+If you have any questions, please feel free to ask us on Slack_.
 
-If you have any questions don't hesitate to ask us on Slack_.
+Reporting Issues
+================
 
-.. _Slack: https://crate.io/docs/support/slackin/
+Before you report an issue, please search the existing issues to make sure
+someone hasn't already reported it.
```

Comment:
can we link this to the organization issue overview page like this: https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+user%3Acrate+